### PR TITLE
Periods managment

### PR DESCRIPTION
**Pull request by Abriscioli**
@andresmoschini 

Hi Andres! I made some changes to allow working with multiple periods. Now you can create evaluations for different periods and also check evaluations by period. Could you review?

### CODE REVIEW

File: source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsApiController.cs [hunk 1]
```diff
@@ -44,9 +44,9 @@ public JsonNetResult GetEmployeesToGenerateEvalution(string period)
 
         [AcceptVerbs(HttpVerbs.Post)]
         [CommonJobsAuthorize(Roles = "EmployeeManagers")]
-        public JsonNetResult GenerateEvalutions(PeriodCreation model)
+        public JsonNetResult GenerateEvalutions(string period, PeriodCreation model)
```
Comment (andresmoschini) [inline source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsApiController.cs]: What do you think about to move `period` inside `PeriodCreation` model?

File: source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js [hunk 2]
```diff
@@ -308,13 +309,13 @@
...
-    getDashboardEvaluations();
+    getDashboardEvaluations(window.ViewData.period);
     ko.applyBindings(viewmodel);
     commonSuggest($('.content-modal .search'), 'UserName');
 
-    function getDashboardEvaluations() {
+    function getDashboardEvaluations(period) {
         viewmodel.isLoading(true);
-        var ajax = $.getJSON("/Evaluations/api/getDashboardEvaluations/", function (model) {
+        var ajax = $.getJSON("/Evaluations/api/getDashboardEvaluations/"+period, function (model) {
```
Comment (andresmoschini) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js]: _(Very, very minor)_ could you add spaces before and after `+`?

```
.../getDashboardEvaluations/" + period, function...
                             ^ ^
```

Comment (Abriscioli) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js]: done

File: source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsApiController.cs (hunk 1 above)
Comment (Abriscioli) [inline source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsApiController.cs]: Yes, it looks better. I'll move it

**Pull request merged**